lib/release_notes_bot_web/controllers/webhook_controller.ex

```diff
@@ -144,13 +170,18 @@ defmodule ReleaseNotesBotWeb.WebhookController do
     end
   end
 
-  defp determine_persistence(%{"release" => release, "action" => action}, project_provider) do
+  defp determine_persistence(
+         %{"release" => release, "action" => action},
+         project_provider,
+         page_info \\ nil
+       ) do
     # TO DO: Persist to all persistence providers
     case Persists.persist(
            build_persistence_title(release),
            release["body"],
            Enum.take(project_provider, 1) |> List.first(),
-           action
+           action,
+           page_info
```

Comment:
Does this change belong in the commit where you introduce `page_info` to the `persist/5` function?

Reply:
This change triggers the new `page_info` arg in the `persist/5` function. I can squash these together, but I felt that it was more readable to split these changes up when I originally putting this PR together.

Reply:
For the sake of atomicness - it may make sense to just squash